policy-library/README.md

```diff
@@ -0,0 +1,87 @@
+# Config Validator Policy Library
```

Comment:
We should remove this, with targeted documentation for how to use with the foundation IMHO